src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,84 @@
+.stop-watch__block {
+  margin: 200px auto;
+}
+
+.stop-watch {
+  position: relative;
+  width: 400px;
+  height: 400px;
+
+  border: 4px solid;
+  border-radius: 50%;
+
+  &__number {
+
+    margin: 0;
+
+    &--12 {
+      position: absolute;
+      left: 50%;
+
+      transform: translateX(-50%);
+    }
+
+    &--3 {
+      position: absolute;
+      left: 100%;
+      top: 50%;
+
+      transform: translate(-100%, -50%);
+    }
+
+    &--6 {
+      position: absolute;
+      left: 50%;
+      top: 100%;
+
+      transform: translate(-50%, -100%);
+    }
+
+    &--9 {
+      position: absolute;
+      top: 50%;
+
+      transform: translate(20%, -50%);
+    }
+  }
+
+  &__clock-hand {
+    position: absolute;
+    left: 50%;
```

Comment:
__clock-hand--seconds maybe better, who knows, maybe in the future you will add hand for hours)